Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftLinuxSysmon.yaml

```diff
@@ -25,40 +25,52 @@ ParserQuery: |
     Syslog | where not(disabled)
     | where SyslogMessage has_all ('<Provider Name="Linux-Sysmon"', '<EventID>3</EventID>')
     | project-away ProcessName, ProcessID
-    | parse SyslogMessage with * '<Data Name="SourceIp">' SrcIpAddr:string '</Data>' *
-    | extend outbound = (SrcIpAddr == HostIP or SrcIpAddr in ('127.0.0.1', '0.0.0.0'))
+    | parse-kv SyslogMessage as (
```

Comment:
No need to use parse-kv for a single parameter. Parse works just as well, and is much more efficient. It also does not need the regex mode in this case.